plugin.video.tv3.cat/addon.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<addon id="plugin.video.tv3.cat" name="TV3cat" version="1.4.8" provider-name="jqandreu">
+    <requires>
+        <import addon="xbmc.python" version="3.0.0"/>
+		<import addon="script.module.beautifulsoup4" version="4.6.3"/>
+        <import addon="script.module.future" version="0.17.1"/>
+    </requires>
+    <extension point="xbmc.python.pluginsource" library="addon.py">
+        <provides>video</provides>
+    </extension>
+    <extension point="xbmc.addon.metadata">
+        <platform>all</platform>
+        <summary lang="en">Entertainment, news, documentaries, etc from catalan television www.tv3.cat</summary>
+        <description lang="en">Entertainment, news, documentaries, etc from catalan television www.tv3.cat. [CR]Featured shows, most viewed, A to Z programs. [CR]Live channels: TV3cat, K33, Super3, News 3/24, Esport3</description>
+        <summary lang="ca">Tots els programes de TV3 a la carta.</summary>
+        <description lang="ca">Tota la programació de TV3 a la carta. [CR]Les llistes dels més vistos, destacats, programes ordenats per ordre alfabètic. [CR]Els canals en directe: TV3cat, K33, Super3, 3/24 i Esport3 </description>
+        <summary lang="es">Toda la programación de TV3 a la carta</summary>
+        <description lang="es">Toda la programación de TV3 a la carta.[CR]Las listas de los más vistos, destacados, programas ordenados de la A a la Z. [CR]Canales en directo: TV3cat, K33, Super3, 3/24, Esport3</description>
+        <language>ca</language>
+        <license>GPL-2.0</license>
+        <source>https://github.com/jqandreu/plugin.video.tv3.cat</source>
+        <website>http://www.tv3.cat</website>
+    </extension>
+    <assets>
```

Comment:
assets is a child of metadata not root/addon

Reply:
You have to delete the assets that is outside of the metadata. Right now you have multiple copies

Reply:
Fixed. Thanks